opta/constants.py

```diff
@@ -24,6 +25,12 @@
 DEV_VERSION = "dev"
 
 SESSION_ID = int(time.time() * 1000)
+UPGRADE_WARNINGS: Dict[str, str] = {
+    "0.21.0": "Performing updates on AWS environments to the network load balancer which will cause a new one to be "
```

Comment:
```suggestion
    "0.21.0": "If you are upgrading an AWS environment to a newer version, this upgrade will cause a 5 min downtime for any public traffic. This will update the network load balancer which will cause a new one to spin up."
```

Reply:
updated with better text